src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php

```diff
@@ -434,11 +434,13 @@ private function modifyCollection($data, $user, $breadcrumb): Collection
 
         $this->em->persist($collectionEntity);
 
+        $isSystemCollection = 2 === $collection->getType()->getId();
+
         if ($isNewLocale) {
             $this->domainEventCollector->collect(
                 new CollectionTranslationAddedEvent($collectionEntity, $collection->getLocale(), $data)
             );
-        } else {
+        } elseif (!$isSystemCollection) {
             $this->domainEventCollector->collect(
```

Comment:
can we add a comment like:
```suggestion
            // do not dispatch modify event for system collections because the current implementation of the SystemCollectionManager 
            // triggers this code for every locale each time the cache of the project is cleared
            $this->domainEventCollector->collect(
```